airflow/ui/src/layouts/Nav/NavButton.tsx

```diff
@@ -31,8 +31,12 @@ type NavButtonProps = {
 } & ButtonProps;
 
 export const NavButton = ({ icon, title, to, ...rest }: NavButtonProps) => (
-  <Button as={RouterLink} to={to} {...navButtonProps} {...rest}>
-    <Box alignSelf="center">{icon}</Box>
-    <Box fontSize="xs">{title}</Box>
-  </Button>
+  <Box as={NavLink} to={to ?? ""}>
```

Comment:
It looks like `to` is optional because there is possibly a `href` instead, but it doesn't appear that the component ever uses `href`?

Reply:
From react-router-dom, it does expect `to` to be a string:

```export type To = string | Partial<Path>;```

Reply:
Yeah, the typing on this component just looks incorrect… `href` and `target` should be removed from `NavButtonProps` and `to` should not be optional.

Reply:
Yeah, I reread your comment and realized that. Cleaned up now.